Add user_id for third-party system to record sessions. (#4206)
infiniflow/ragflow: docs/references/http_api_reference.md

  The ID of the associated chat assistant.
- `"name"`: (*Body parameter*), `string`  
  The name of the chat session to create.
- `"user_id"`: (*Body parameter*), `string`  

>   The user ID of third-party upstream system which invokes the API. It's optional.

Reviewer: ```suggestion
  Optional user-defined ID.
```

  The ID of the session to update.
- `"name"`: (*Body Parameter*), `string`  
  The revised name of the session.
- `"user_id"`: (*Body parameter*), `string`  

>   The user ID of third-party upstream system which invokes the API. It's optional.

Reviewer: ```suggestion
  Optional user-defined ID.
```

  The name of the chat session to retrieve.
- `id`: (*Filter parameter*), `string`  
  The ID of the chat session to retrieve.
- `user_id`: (*Filter parameter*), `string`  

>   The user ID of third-party upstream system which updates this field by creating session.

Reviewer: ```suggestion
  The optional user-defined ID passed in when creating session.
```

  - `false`: Disable streaming.
- `"session_id"`: (*Body Parameter*)  
  The ID of session. If it is not provided, a new session will be generated.
- `"user_id"`: (*Body parameter*), `string`  

>   The user ID of third-party upstream system which invokes the API. It's optional and only needed when no session_id provided.

Reviewer: ```suggestion
  The optional user-defined ID. Valid *only* when no `session_id` is provided.
```

- Body:
  - the required parameters:`str`
  - the optional parameters:`str`
    - `"user_id"`: `string`  

>       The user ID of third-party upstream system which invokes the API. It's optional.

Reviewer: ```suggestion
      The optional user-defined ID.
```

  - `false`: Disable streaming.
- `"session_id"`: (*Body Parameter*)  
  The ID of the session. If it is not provided, a new session will be generated.
- `"user_id"`: (*Body parameter*), `string`  

>   The user ID of third-party upstream system which invokes the API. It's optional and only needed when no session_id provided.

Reviewer: ```suggestion
  The optional user-defined ID. Valid *only* when no `session_id` is provided.
```

  Indicates whether the retrieved sessions should be sorted in descending order. Defaults to `true`.
- `id`: (*Filter parameter*), `string`  
  The ID of the agent session to retrieve.
- `user_id`: (*Filter parameter*), `string`  

>   The user ID of third-party upstream system which updates this field by creating session.

Reviewer: ```suggestion
  The optional user-defined ID passed in when creating session.
```
